SignNow.Net/Interfaces/IUserService.cs

```diff
@@ -49,5 +50,14 @@ public interface IUserService
         /// <param name="cancellationToken">Propagates notification that operations should be canceled</param>
         /// <returns></returns>
         Task SendPasswordResetLinkAsync(string email, CancellationToken cancellationToken = default);
+
+
+        /// <summary>
+        /// Returns an enumerable of user's documents that have been modified (added fields, texts, signatures, etc.) in descending order by modified date
+        /// </summary>
+        /// <param name="perPage">How many document objects to display per page in response. By default, it�s 15, maximum 100.</param>
+        /// <param name="cancellationToken">Propagates notification that operations should be canceled.</param>
+        /// <returns></returns>
+        Task<IEnumerable<SignNowDocument>> GetModifiedDocumentsForUser(int? perPage = 15, CancellationToken cancellationToken = default);
```

Comment:
perPage cannot be null, less than 0 and more than 100. There is must be validation before we making request.